Probability_of_Superiority

```diff
@@ -0,0 +1,39 @@
+psup <- function(d=NULL, CL=NULL, plot=F, AUC=F) {
+  if (is.null(CL)) {
```

Comment:
Co kdyz budou `NULL` oba argumenty? :) Tak to vyhori, ikdyz tu mas kontrolu na rozsah CL, takze evidentne se nechces spolihat na to, ze user udela vzdycky vsechno bezchybne. Pridal bych kontrolu ze alespon jeden z argumentu `!is.null()`:)